### DISCUSSION TITLE
Welcome to chans_jupyter Discussions!

### DISCUSSION THREAD

**Discussion by goodboychan** [goodboychan/goodboychan.github.io#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by QuentinDvt** [comment]
Hello, How were you able to obtain the subset of vae celaba data in numpy format?

**Reply by goodboychan** [reply]
Hi,
Actually, Coursera already made it for the assignment. I just downloaded it.

**Comment by MSBAshadi** [comment]
hello, from where did you get dots csv file? in the notebook "improving your model performance"

**Reply by goodboychan** [reply]
Sorry for late reply.
I can share the data if you give a mail address.
thanks.

**Comment by JoeySams** [comment]
How do we find the csv files that were used in this code? I was not able to find any in DataCamp. I chose this course for a project for a class and I really need the code to be able to work in a Jupyter notebook, but I have no files for the code to upload.
